src/hiccup/page.clj

```diff
@@ -78,7 +78,11 @@
   "Include a list of external javascript files."
   [& scripts]
   (for [script scripts]
-    [:script {:type "text/javascript", :src (util/to-uri script)}]))
+    (if (:src script)
```

Comment:
Would a `map?` be better here?

Reply:
As long as there are no concerns about possibly `nil` being fed to the uri converter, in which case I can make it `if-let`

Reply:
If `nil` is passed, it's probably better to throw an error than to silently ignore it.